AudibleBoulders/AudibleBoulders
# Update readme and improve style convention adherence

#### What's this PR do?
Updates README.md's server and client routes info. Enforces naming conventions in /api/repos request handlers. Removes :githubId param from GET /api/setup/... since it's in the cookies.

#### What are the important parts of the code?
README.md

#### How should this be tested by the reviewer?
Pull down, run, make sure everything still works as expected, particularly the #/add page and setup page.

#### Is any other information necessary to understand this?
Removing :githubId from the route params meant some related small changes in the client side.

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
#### Screenshots (if appropriate)
